src/aio.rs

```diff
@@ -0,0 +1,265 @@
+// Copyright (C) 2019 Alibaba Cloud Computing. All rights reserved.
```

Comment:
Just nit: It seems that the commit message needs to be modified as "... may be used by some ..."